chef_master/source/resource_remote_file.rst

```diff
@@ -68,6 +71,7 @@ where
 * ``remote_file`` is the resource
 * ``name`` is the name of the resource block
 * ``action`` identifies the steps the chef-client will take to bring the node into the desired state
+* ``remote_user``, ``remote_password`` and ``remote_domain`` are used for accessing a remote UNC path on Windows. These properties are Windows specific only.
```

Comment:
* ``remote_user``, ``remote_password``, and ``remote_domain`` are used to authenticate to a remote server when accessing a file over SMB/CIFS specified as a UNC path. This is only supported on Windows systems.